src/Tools/BuildValidator/CompilationDiff.cs

```diff
@@ -141,6 +146,26 @@ private CompilationDiff(ImmutableArray<Diagnostic> diagnostics, string originalP
                         writeVisualization(originalPeMdvPath, optionsReader.PeReader.GetMetadataReader());
                         writeVisualization(originalPdbMdvPath, optionsReader.PdbReader);
 
+                        var originalPdbXmlPath = Path.Combine(originalPath, assemblyName + ".pdb.xml");
```

Comment:
This didn't end up helping in practice, but using pdb2xml was an attempt to determine if PDB differences were "significant" versus being merely differences of the offsets of various resources in the assembly.

It could help in the future, so I'd like to leave it here unless it introduces a problem.